### PR TITLE
Intro: Add `jax.vmap` section

### PR DESCRIPTION
**Pull request by Smit-create**
https://github.com/QuantEcon/lecture-jax/pull/87#issuecomment-1667002764

### CODE REVIEW

File: lectures/jax_intro.md [hunk 1]
```diff
@@ -424,8 +425,77 @@ ax.legend(loc='upper center')
...
 
+## Vectorizing map
+
+JAX has the function [jax.vmap](https://jax.readthedocs.io/en/latest/_autosummary/jax.vmap.html) for creating a vectorized
+function of the given function argument over some axis.
+
+Consider the following function
+
+$$
+     f(x, y)= \frac{cos(x^2 + y^2)}{(1 + x^2 + y^2)}
```
Comment (jstac) [inline lectures/jax_intro.md:436]: `cos` should be `\cos`

File: lectures/jax_intro.md [hunk 2]
```diff
@@ -424,8 +425,77 @@ ax.legend(loc='upper center')
...
+$$
+
+Assume that $x$ and $y$ are 2-D.
```
Comment (jstac) [inline lectures/jax_intro.md:439]: Please cut this line. ("Assume...")

**Comment by Smit-create** [comment]
> Can you think of one?
> 
> Also, when you first explained this double vmap idea to me, you used a really simple example in one of your recent PRs --- I think the one about inventories. (You compared two different types of vmap and how they worked relative to a loop.) Could you please add that in too, at the start of the discussion.

I updated it with a very simple example, and now it's easy for newcomers to understand this.

> I think it would be better if you got rid of f_loops and showed how it would be possible to create a vectorized version of f that accepts flat arrays x and y.

I removed that example completely as broadcasting is very easy to understand, but we can compare loops and vectorized implementation when `f` isn't broadcastable as updated in the last commit.

**Comment by jstac** [comment]
Beautiful, thanks @Smit-create !

**Pull request merged**